TCC.Lib/Options/CompressionAlgo.cs

```diff
@@ -1,9 +1,29 @@
-﻿namespace TCC.Lib.Options
+﻿using System;
+
+namespace TCC.Lib.Options
 {
     public enum CompressionAlgo
     {
         Lz4,
         Brotli,
         Zstd
     }
+
+    [Flags]
+    public enum BenchmarkCompressionAlgo
+    {
+        Lz4 = 1 << 0,
+        Brotli = 1 << 1,
+        Zstd = 1 << 2,
+        All = Lz4 | Brotli | Zstd
+    }
+
+    [Flags]
+    public enum BenchmarkContent
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this enumeration to match the regular expression: '^([A-Z]{1,3}[a-z0-9]+)*([A-Z]{2})?s$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2342)